### PR TITLE
Add preference in toolbar

### PR DESCRIPTION
**Pull request by MathuraMG**
Below are the current changes included to add preferences
screen shot 2016-06-20 at 3 22 48 pm
screen shot 2016-06-20 at 3 22 56 pm

### CODE REVIEW

**Comment by catarak** [comment]
I didn't realize you'd be working on this so soon! Can you do me a favor and hold back on other changes? I'm currently in the process of doing some cleaning up--including redoing the file structure and adding ESLint. This has some ES5 syntax which is totally fine, because I haven't been writing 100% perfect ES6, but I need to go through and fix some of this stuff.

File: shared/components/Preferences/Preferences.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+class Preferences extends React.Component {
+	render() {
+		let preferencesContainerClass = classNames({
+			"preferences": true,
+			"preferences--selected": this.props.isPreferencesShowing
+		});
+		return (
+			<div className={preferencesContainerClass} tabindex="0">
+				<div className="preferences__heading">
+					<h2 className="preferences__title">Preferences</h2>
```
Comment (catarak) [inline shared/components/Preferences/Preferences.jsx]: This heading doesn't need to be inside of a div

File: styles/components/_preferences.scss [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	line-height: #{44 / $base-font-size}rem;
+}
+
+.preference {
+	margin: 0 #{20 / $base-font-size}rem;
+}
+
+.preference__title {
+	margin: 0 0;
+	font-size: #{$base-font-size}px;
```
Comment (catarak) [inline styles/components/_preferences.scss]: this font size should be in rem's

File: styles/components/_preferences.scss [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+.preferences__heading {
+	display: flex;
+	flex-direction: rows;
+	margin-left: #{20 / $base-font-size}rem;
+}
+.preferences__title {
+	margin: 0 0;
+	font-size: #{$menu-font-size}px;
```
Comment (catarak) [inline styles/components/_preferences.scss]: this font size should be in rem's

File: styles/layout/_ide.scss [hunk 4]
```diff
@@ -16,4 +16,11 @@
 
 .toolbar {
 	width: 100%;
-}
\ No newline at end of file
+}
+
+.preferences {
+	width: 22.5%;
```
Comment (catarak) [inline styles/layout/_ide.scss]: this should be a fixed width or have a min/max width value

**Comment by catarak** [comment]
I'm just going to make these css changes myself, so that I can get started on file restructure.

**Comment by MathuraMG** [comment]
My bad, I just saw this! Sure, I'll hold back on the rest.

**Pull request merged**